src/main/java/com/digirati/taxonomy/manager/lookup/model/Match.java

```diff
@@ -0,0 +1,48 @@
+package com.digirati.taxonomy.manager.lookup.model;
+
+import org.ahocorasick.trie.Emit;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Represents a pairing between the details of where a term was found in a piece of input text, and
+ * what that term could mean.
+ */
+public class Match {
+
+    private final Emit emit;
+
+    private final List<Concept> matchingConcepts;
+
+    public Match(Emit emit, List<Concept> matchingConcepts) {
```

Comment:
You use `java.util.Collection` in most other places, but have a `java.util.List` here. Any particular reason why?

Reply:
The reason here is just that I happened to use `List` in the `ConceptLookupTable`. I don't see any reason why that has to be a `List` there so I'm happy to change it

Reply:
Unless you do need an indexed and ordered collection of items, returning `Collection` is more appropriate.